### DISCUSSION TITLE
Displaying videos on Pavlovia using jsPsych

### DISCUSSION THREAD

**Discussion by natnova** [jspsych/jsPsych#1106] [Q&A]
I have been working on a trial where I play a video and get the user's button response. Currently, I am using the video button plugin and it worked perfectly on my local server. I uploaded my trial to pavlovia and now the video itself is no longer appearing. I have tried using both mp4 and mov files. The videos are begin stored in a folder labeled video in my repository, and this is what happens when the video should be getting played. I get no errors on my console and everything shows up other than the video itself. 

![Screen Shot 2020-10-04 at 2 27 58 PM](https://user-images.githubusercontent.com/55930940/95116756-83b02980-0715-11eb-89dd-bb405e5309d7.png)

Here is my git lab repo: https://gitlab.pavlovia.org/lclab2/combotrial
I'm not sure if something is missing in my code in order to add the videos properly but I would love some input. 
Thanks!

**Comment by jodeleeuw** [comment]
Is there a link to the live experiment available?
